dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/config/validation/AbstractConfigurationValidator.java

```diff
@@ -1004,8 +1017,10 @@ private boolean hasFeature(ConfigurationFile file, String featureName) {
      */
     private ValidationContext getCommonParent(ValidationContext context, String featureName) {
         ValidationContext toplevl = context;
-        while (!hasFeature(toplevl.getConfigFile(), featureName)) {
-            toplevl = toplevl.getParent();
+        if (toplevl.getConfigFile() != null) { // Protect against null.
```

Comment:
Its only protecting against null the first time through the loop, what about the other times?